microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -836,6 +867,34 @@ This feature is currently disabled by default in the browser.  To enable this fe
 
 ---
 
+
+<!-- ------------------------------ -->
+#### Enable or disable the browser responding to accelerator keys (shortcut keys) (incoming 3)
+
+`ICoreWebView2AcceleratorKeyPressedEventArgs` has a `IsBrowserAcceleratorKeyEnabled` property to allow you to control whether the browser handles accelerator keys (shortcut keys), such as **Ctrl+P** or **F3**.
+
+See also:
+* [Keyboard accelerators](#enable-or-disable-the-browser-responding-to-accelerator-keys-shortcut-keys-incoming-3), below.
```

Comment:
I think this link might be a mistake. It links back to this section.

I'm assuming that you are intending to link it to a documentation about keyboard accelerators.

Reply:
fixed link
`* [Keyboard accelerators](#keyboard-accelerators), below.`